### PR TITLE
Homework5: Miron Chuduk

### PR DESCRIPTION
**Pull request by MironChuduk**
### PR checklist
Please make sure you've passed PR checklist
- [x] my PR passes CI checks (flake8 and pycodestyle).
- [x] my PR named according to guidelines.
- [x] my PR contains complete tasks descriptions for all tasks.
- [x] my PR made from the fork feature branch to appropriate branch.
- [x] my PR contains only code that solves appropriate homework tasks. Redundant 
files (configs, etc. ) are put to .gitignore.

### CODE REVIEW

File: src/homework5/task2.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+i = 0
+list_of_counties = []
+country = ""
+while i < number_of_cities:
+    city_for_search = input(f"Введите город №{i + 1}: ")
+    for country in dict_of_names:
+        for city in dict_of_names.get(country):
+            if city == city_for_search:
+                list_of_counties.append(country)
+                break
```
Comment (Shurke) [inline src/homework5/task2.py]: А почему не использовать оператор `in` для проверки вхождения города в список городов?

File: src/homework5/task6.py [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+
+import string
+
+my_string = input("Введите строку:\n")
+for symbol in my_string:
+    if symbol in string.punctuation:
+        my_string = my_string.replace(symbol, "")
+my_list = my_string.split()
+print(f"Всего слов: {len(my_list)}")
```
Comment (Shurke) [inline src/homework5/task6.py]: Не учтено условие про различные слова.

**Review by Shurke — CHANGES_REQUESTED** [review]

File: src/homework5/task2.py (hunk 1 above)
Comment (nlifeyouth) [reply src/homework5/task2.py]: Исправлено

File: src/homework5/task6.py (hunk 2 above)
Comment (nlifeyouth) [reply src/homework5/task6.py]: Исправлено

**Review by Shurke — COMMENTED** [review]
Итоговая оценка - 8.

**Pull request closed without merging**